src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -66,8 +75,12 @@ type State = {
     file?: BoxItem,
     accessStats?: FileAccessStats,
     versions?: FileVersions,
+    comments?: Comments,
```

Comment:
It looks like we're always setting it but it may be undefined. would that be `comments: ?Comments`

Reply:
I think the way I have it is correct. It can possibly be undefined, or not be in the state at all. See https://flow.org/en/docs/types/primitives/#toc-optional-object-properties